src/devices/bus/coco/coco_ssc.cpp

```diff
@@ -92,16 +92,20 @@ namespace
 	protected:
 		// device-level overrides
 		virtual void device_start() override;
+		virtual void device_timer(emu_timer &timer, device_timer_id id, int param, void *ptr) override;
 		virtual DECLARE_READ8_MEMBER(ff7d_read);
 		virtual DECLARE_WRITE8_MEMBER(ff7d_write);
 		virtual void set_sound_enable(bool sound_enable) override;
+        static constexpr int BUSY_TIMER_ID  = 0;
```

Comment:
You should use `device_timer_id` for constants of that type rather than `int` - they aren't guaranteed to be the same.